content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md

```diff
@@ -28,15 +60,9 @@ Let's pick one image from this list and verify its signature using
 the `cosign verify` command:
 
 ```shell
-COSIGN_EXPERIMENTAL=1 cosign verify registry.k8s.io/kube-apiserver-amd64:v1.24.0
```

Comment:
Wasn't `COSIGN_EXPERIMENTAL` dropped just recently? Is there a new `cosign` release that doesn't require it? Also, should we mention the minimum `cosign` version in the requirements?

Reply:
Yeah, I think we can just re-add it to ensure it works for each version.